setup.cfg

```diff
@@ -1,16 +1,79 @@
+[metadata]
+name = nlp_primitives
+version = attr: nlp_primitives.version.__version__
+author = Alteryx Inc.
+license = BSD 3-clause
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/alteryx/nlp_primitives
```

Comment:
Maybe we should update the URL's like in Featuretools to include some other links?

https://github.com/alteryx/featuretools/blob/54c2ba8a80193f0b05f271fd493fd7c90fc34f82/setup.cfg#L6